NSSNamespace/Initial_Bangazon_Site
# fixed broken things

Fixed some styling things in ```Cart.cshtml``` that were broken. 

Noteable changes:
- Buttons and dropdown in cart view are be properly aligned again.
- Order total is now pushed to the bottom of the page, except on mobile view sizes; in which case it comes after the list of products but before the buttons.

## To Test
- ```git fetch```
- ``` git checkout ew-MoreStyling```
- select a **Bangazon** customer that has items in their cart
- navigate to the customer's cart and make sure the changes listed above are in place
